packages/code-du-travail-api/.gitignore

```diff
@@ -63,8 +63,5 @@ typings/
 # Yarn Integrity file
 .yarn-integrity
 
-# dotenv environment variables file
-.env
```

Comment:
Same should be done in `/packages/code-du-travail-css/.gitignore` ?

Reply:
agree, since code-du-travail-css was not using a env file, I forget to update it :)